Makefile

```diff
@@ -27,11 +27,21 @@ ifeq ($(BUILDKIT_PROGRESS),)
 export BUILDKIT_PROGRESS := plain
 endif
 
-PREEMPT_RT_SUPPORT_ARG := ""
-ifeq ($(PREEMPT_RT),1)
-  PREEMPT_RT_SUPPORT_ARG := "--preempt-rt-support"
-endif
-KERNEL_BUILD_YML_TOOL := tools/kernel-build-yml.sh $(PREEMPT_RT_SUPPORT_ARG)
+#set KERNEL_FLAVOR to PLATFORM
+PLATFORM?=generic
```

Comment:
@mikem-zed , you need to handle PLATFORM for ARM devices.... so far we might have the following platforms:

imx8mp_pollux
imx8mp_epc_r3720
imx8mq_evk

These should point to the https://github.com/lf-edge/eve-kernel/tree/eve-kernel-arm64-v6.1.38-generic branch

Soon we will have the nvidia, or probably nvidia-lenovo-se70, that should point to nvidia's kernel, so I don't think we should handle these platforms in the Makefile... we have a script under tools/prepare-platform.sh that process the platform and copy some files when needed... maybe you can add a parameter to return the corresponding kernel branch according to the platform, so in the Makefile you just call the script.... in this way every time we add a new platform, we change only the "platform" script and not the Makefile...

Reply:
or you could make a new script to handle all kernel branches according to the platform, I think it would make more sense...

Reply:
@rene what needs to be changed in this PR to make it usable (now or later) for ARM builds?